<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
<test-results name="C:\Users\Nathan\Desktop\Training Programs\Coding\Internet Bookstore\Unit Test\Win32\Debug\UnitTest.exe" total="12" errors="0" failures="0" ignored="0" inconclusive="0" not-run="0" skipped="0" invalid="0" date="2023-08-22" time="0.070">
  <culture-info current-culture="en" current-uiculture="en" />
  <test-suite type="Assembly" name="UnitTest.exe" executed="true" result="Success" success="True" time="0.070" asserts="0">
    <results>
        <test-suite type="Namespace" name="BookServiceTestWithStub" executed="true" result="Success" success="True" time="0.070" asserts="0" >
          <results>
              <test-suite type="Fixture" name="TBookServiceTest" executed="True" result="Success" success="True" time="0.070" >
                <results>
                  <test-case name="CreateBook_TitleIsEmpty_RaiseException" executed="True" result="Success" success="True" time="0.000" asserts="0" />
                  <test-case name="CreateBook_SynopsisIsEmpty_RaiseException" executed="True" result="Success" success="True" time="0.000" asserts="0" />
                  <test-case name="CreateBook_SuccessfullyInsert_WillNotRaiseException" executed="True" result="Success" success="True" time="0.000" asserts="0" />
                  <test-case name="GetBook_InvalidIdRange_RaiseException.ZeroID" executed="True" result="Success" success="True" time="0.001" asserts="0" />
                  <test-case name="GetBook_InvalidIdRange_RaiseException.NegativeID" executed="True" result="Success" success="True" time="0.000" asserts="0" />
                  <test-case name="GetBook_InvalidIdRange_RaiseException.ExceedingID" executed="True" result="Success" success="True" time="0.001" asserts="0" />
                  <test-case name="GetBook_MatchingErrorMessage_True.ZeroIdMessage" executed="True" result="Success" success="True" time="0.000" asserts="0" />
                  <test-case name="GetBook_MatchingErrorMessage_True.NegativeIdMessage" executed="True" result="Success" success="True" time="0.000" asserts="0" />
                  <test-case name="GetBook_MatchingErrorMessage_True.ExceedingIdMessage" executed="True" result="Success" success="True" time="0.000" asserts="0" />
                  <test-case name="GetBook_SucessfullyRetrieveBook_True" executed="True" result="Success" success="True" time="0.000" asserts="0" />
                  <test-case name="GetAllBooks_DatabaseHasData_True" executed="True" result="Success" success="True" time="0.000" asserts="0" />
                  <test-case name="GetAllBooks_DataBaseIsEmpty_True" executed="True" result="Success" success="True" time="0.000" asserts="0" />
                </results>
              </test-suite>
          </results>
        </test-suite>
    </results>
  </test-suite>
</test-results>
